docs/csharp/tutorials/index.md

```diff
@@ -14,14 +14,14 @@ ms.assetid: fcc83b5b-fb68-4e48-9132-0882677d8056
 
 # C# Tutorials
 
-These exercises enable you to build C# programs using core CLR.
+These tutorials enable you to build C# programs using [CoreCLR](https://github.com/dotnet/coreclr).
```

Comment:
Wondering if it would be even better to say:"

The following tutorials enable you to build C# programs using [CoreCLR](https://github.com/dotnet/coreclr):